dist/testbed-support/makefile.iotlab.single.inc.mk

```diff
@@ -65,13 +65,15 @@ _IOTLAB_EXP_ID := $(if $(IOTLAB_EXP_ID),--id $(IOTLAB_EXP_ID))
 IOTLAB_NODE_AUTO_NUM ?= 1
 
 # board-archi mapping
-IOTLAB_ARCHI_iotlab-m3      = m3:at86rf231
+IOTLAB_ARCHI_arduino-zero   = arduino-zero:xbee
+IOTLAB_ARCHI_b-l072z-lrwan1 = st-lrwan1:sx1276
```

Comment:
You can also add the line:
```mk
IOTLAB_ARCHI_b-l475e-iot01a = st-iotnode:multi
```

Reply:
Oh I did not understood it was this one :D
Is this one not supposed to be a lora board and so I should have been in the lora group to use it ?

I will fix and directly amend the commit.
I also stopped my experiment so you can test.

Reply:
>Is this one not supposed to be a lora board and so I should have been in the lora group to use it ?

No it's public, for now. But indeed, it has a sub-ghz radio (not supported by RIOT though...).